search-resources-api/src/main/java/no/unit/nva/dynamodb/IndexDocumentGenerator.java

```diff
@@ -175,22 +180,32 @@ private static IndexPublisher extractPublisher(JsonNode record) {
     }
 
     private static Instant extractModifiedDate(JsonNode record, UUID id) {
-        var modifiedDate = Instant.parse(textFromNode(record, MODIFIED_DATE_JSON_POINTER));
-        if (isNull(modifiedDate)) {
-            logMissingField(id, MODIFIED_DATE);
-        }
-        return modifiedDate;
+        return getInstant(record, id, MODIFIED_DATE_JSON_POINTER, MODIFIED_DATE);
     }
 
     private static Instant extractPublishedDate(JsonNode record, UUID id) {
-        var publishedDate = Instant.parse(textFromNode(record, PUBLISHED_DATE_JSON_POINTER));
-        if (isNull(publishedDate)) {
-            logMissingField(id, PUBLISHED_DATE);
+        return getInstant(record, id, PUBLISHED_DATE_JSON_POINTER, PUBLISHED_DATE);
+    }
+
+    private static Instant getInstant(JsonNode record, UUID id, String fieldJsonPtr, String fieldName) {
+        String textFromNode = textFromNode(record, fieldJsonPtr);
+        if (isNull(textFromNode) || textFromNode.isBlank()) {
```

Comment:
We use the commons library, so use the relevant method :)

Reply:
Knowledge of StringUtils is updated...:-/